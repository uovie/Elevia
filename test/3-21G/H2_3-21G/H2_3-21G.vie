RHF 3-21G 2 0 1

 H                  0.00000000    0.00000000    0.35611500
 H                  0.00000000    0.00000000   -0.35611500