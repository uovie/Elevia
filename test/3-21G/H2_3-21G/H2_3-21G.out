
General Infomation
Method: RHF, Basis Set: 3-21G, Total Num: 2, Charge: 0, Spin Multi: 1.

Nuclear repulsion energy:
V_{nn} = 0.74298641

The number of atomic orbitals is 4.

Overlap Integrals:
S =
1.00000000 0.64589894 0.43095120 0.49109724
0.64589894 1.00000000 0.49109724 0.84710881
0.43095120 0.49109724 1.00000000 0.64589894
0.49109724 0.84710881 0.64589894 1.00000000

Kinetic-Energy Integrals:
T_{e} =
1.54940455 0.29315177 0.25003416 0.18204678
0.29315177 0.27478800 0.18204678 0.20702633
0.25003416 0.18204678 1.54940455 0.29315177
0.18204678 0.20702633 0.29315177 0.27478800

Nuclear Attraction Integrals:
V_{ne} =
-2.45507849 -1.23704139 -1.04823955 -0.97517233
-1.23704139 -1.24078875 -0.97517233 -1.09622061
-1.04823955 -0.97517233 -2.45507849 -1.23704139
-0.97517233 -1.09622061 -1.23704139 -1.24078875

Core Hamiltonian:
H_{core} =
-0.90567394 -0.94388962 -0.79820539 -0.79312555
-0.94388962 -0.96600075 -0.79312555 -0.88919428
-0.79820539 -0.79312555 -0.90567394 -0.94388962
-0.79312555 -0.88919428 -0.94388962 -0.96600075

Initial Coefficient Matrix:
C =
-0.40161430 -0.34315969  0.72293623  1.04638102
-0.20759728 -1.37090364 -0.69766978 -1.62298828
-0.40161430  0.34315969  0.72293623 -1.04638102
-0.20759728  1.37090364 -0.69766978  1.62298828

Initial Density Matrix:
P =
0.16129405 0.08337404 0.16129405 0.08337404
0.08337404 0.04309663 0.08337404 0.04309663
0.16129405 0.08337404 0.16129405 0.08337404
0.08337404 0.04309663 0.08337404 0.04309663

Fock Matrix:
F =
 0.11309098 -0.47995411 -0.55636828 -0.50280146
-0.47995411 -0.28812556 -0.50280146 -0.33623830
-0.55636828 -0.50280146  0.11309098 -0.47995411
-0.50280146 -0.33623830 -0.47995411 -0.28812556


SCF Procedure:
Iter       E_{elec}            E_{tot}             \Delta E              RMSD        Time(s)
01     -1.817477122095     -1.074490711919     -1.817477122095      0.155562905715   0.00036
02     -1.859220122232     -1.116233712056     -0.041743000137      0.022187532885   0.00026
03     -1.864657868661     -1.121671458484     -0.005437746428      0.003185089757   0.00028
04     -1.865423143894     -1.122436733717     -0.000765275233      0.000456327527   0.00026
05     -1.865532455710     -1.122546045534     -0.000109311816      0.000065354980   0.00029
06     -1.865548104496     -1.122561694320     -0.000015648786      0.000009359619   0.00027
07     -1.865550345451     -1.122563935274     -0.000002240955      0.000001340400   0.00027
08     -1.865550666377     -1.122564256201     -0.000000320926      0.000000191960   0.00027
09     -1.865550712337     -1.122564302161     -0.000000045960      0.000000027491   0.00028
10     -1.865550718919     -1.122564308743     -0.000000006582      0.000000003937   0.00027
11     -1.865550719862     -1.122564309686     -0.000000000943      0.000000000564   0.00026

**Hartree-Fock energy: HF = -1.12256431

Normal termination. Congratulations!
